src/win/util.c

```diff
@@ -175,9 +172,6 @@ int uv_cwd(char* buffer, size_t* size) {
     return uv_translate_sys_error(GetLastError());
   }
 
-  /* utf16_len contains the length, *not* including the terminating null. */
```

Comment:
It contains the length, including the null, but is also outside of the bounds of the input array